src/main/java/net/neoforged/neoforge/common/data/internal/NeoForgeLanguageProvider.java

```diff
@@ -74,6 +74,7 @@ protected void addTranslations() {
         add(Tags.Blocks.OBSIDIANS, "Obsidians");
         add(Tags.Blocks.OBSIDIANS_NORMAL, "Normal Obsidians");
         add(Tags.Blocks.OBSIDIANS_CRYING, "Crying Obsidians");
+        add(Tags.Blocks.PUMPKINS, "Pumpkins");
```

Comment:
Why is this all the way up here and not down below with the other pumpkin entries?